ansible/roles/products/common/resources/templates/scripts/launch-local-website.j2

```diff
@@ -0,0 +1,22 @@
+#jinja2: trim_blocks:False
+#! /bin/bash
+HOST='localhost'
+PORT={{ local_website_port }}
+
+if [ ! "$(docker ps -q -f name={{ container_name  }})" ]; then
```

Comment:
why this? If its not running then display an error message instead of starting the container. Also waiting 3 seconds with no message displayed wont look good

Reply:
ok, fixed in the last commit